### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS3.2.1][Eclipse][Wipro]:Combo box is closed automatically while navigating to the list items  in 'Container Registry' combo box in push image window

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1450]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. Push image dialog will open, Tab to "container Registry" combo box and press  Alt+down arrow and open combo box.
5. Press down arrow  to select the list items and observe the combo box behavior

**Expected Result:**
Using down arrow, list item should be selected and should not close the dropdown.

**Actual Result:**
Drop down is getting closed with down arrow.
Container registry is automatically selected as value

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![drop down is getting closed with down arrow in push image window](https://user-images.githubusercontent.com/22956130/35730325-b7a1046a-0837-11e8-828d-f98e213d8f6c.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
under further investigation, to see wherher there's workaround.

**Comment by Eskibear** [comment]
fixed in #1600

**Comment by WiproPSAccTest** [comment]
Verified the issue in V 3.6.0.
Issue still exists

**Comment by Eskibear** [comment]
Strange, attaching my test, it works well.
![eclipse_dropdown](https://user-images.githubusercontent.com/2351748/39558109-a8c4f668-4ebe-11e8-82d3-10955b45515e.gif)

**Comment by WiproPSAccTest** [comment]
Verified the issue in V 3.6.0.
Issue still exists
![1450-not fixed](https://user-images.githubusercontent.com/22956130/39629285-46be5570-4fc9-11e8-93f7-bd39abafb2b0.jpg)
 Please find the screenshot

**Comment by WiproPSAccTest** [comment]
Drop down is getting closed with down arrow.
Container registry is automatically selected as value

**Comment by WiproPSAccTest** [comment]
Verified in V3.6.0 Issue got fixed .

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749889
